CONTRIBUTING.md

```diff
@@ -1 +1,56 @@
-TODO
+# Contributing to Hiero
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued.
+See the Table of Contents for different ways to help and details about how this project handles them.
```

Comment:
I didn't see a Table of Contents in this doc. But there are different sections. I'm guessing this language was created by the contributing-gen?